pure.zsh

```diff
@@ -268,8 +284,10 @@ prompt_pure_setup() {
 
 	zstyle ':vcs_info:*' enable git
 	zstyle ':vcs_info:*' use-simple true
-	zstyle ':vcs_info:git*' formats ' %b'
-	zstyle ':vcs_info:git*' actionformats ' %b|%a'
+	zstyle ':vcs_info:*' max-exports 2
+	# vcs_info_msg_0_ = brach, vcs_info_msg_1_ = git top level (x prefixed)
+	zstyle ':vcs_info:git*' formats ' %b' 'x%R'
```

Comment:
Absolutely, would be confusing if you didn't look in the correct place otherwise! :)